src/shared/components/markdown/markdown.component.spec.ts

```diff
@@ -1,11 +1,12 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
-import { DebugElement } from '@angular/core';
+import { Component, DebugElement, ViewChild } from '@angular/core';
 import { MarkdownComponent as CCDMarkDownComponent } from './markdown.component';
 import { NgxMdModule, NgxMdComponent } from 'ngx-md';
 import { By } from '@angular/platform-browser';
 import { PipesModule } from '../../pipes';
+import { ConvertHrefToRouterService } from '../case-editor/services';
```

Comment:
Imports not correct (not because of changes but would be good to clean up if you have time)